airflow/decorators/__init__.pyi

```diff
@@ -126,7 +126,25 @@ class TaskDecoratorCollection:
             such as transmission a large amount of XCom to TaskAPI.
         """
     @overload
-    def virtualenv(self, python_callable: Function) -> Function: ...
+    def virtualenv(self, python_callable: F) -> F: ...
+    @overload
+    def kubernetes(
+        python_callable: Optional[Callable] = None, multiple_outputs: Optional[bool] = None, **kwargs
+    ) -> TaskDecorator:
```

Comment:
This is wrong; again, see the documentation and the Docker decorator.

Reply:
Moved, thanks